src/pages/instances.tsx

```diff
@@ -50,6 +50,12 @@ export const Instances = () => {
 
     return (
         <div className="m-5 flex flex-col h-screen"><h1 className="text-4xl font-bold mb-5">Scanned instances</h1>
+            <p>
+                This is an overview of all scanned Etherpad instances. You can start a new scan by entering the url of
+                the instance and clicking on the "Start scan" button.
```

Comment:
We could wrap Start Scan in a pre and set a different background like lightgrey.

Reply:
Done